apps/examples/testcase/le_tc/kernel/tc_libc_misc.c

```diff
@@ -335,72 +334,73 @@ static void tc_libc_misc_lldbg(void)
 
 	TC_SUCCESS_RESULT();
 }
+#endif /* !CONFIG_BUILD_PROTECTED && CONFIG_ARCH_LOWPUTC */
+#endif /* CONFIG_DEBUG_ERROR */
 
+#ifdef CONFIG_DEBUG_VERBOSE
 /**
- * @fn                  :tc_libc_misc_llvdbg
+ * @fn                  :tc_libc_misc_vdbg
  * @brief               :This is intended for general debug output that is normally suppressed.
  * @scenario            :When debug out is required with all details
- * @API's covered       :llvdbg
+ * @API's covered       :vdbg
  * @Preconditions       :None
  * @Postconditions      :None
  * @Return              :void
  */
-static void tc_libc_misc_llvdbg(void)
+
+static void tc_libc_misc_vdbg(void)
 {
 	int ret_chk = -1;
-	const char *msg_output = "tc_libc_misc_llvdbg: Examples";
+	const char *msg_output = "tc_libc_misc_vdbg: Examples";
 	/*
-	   in debug.h file llvdbg is defined as :
+	   in debug.h file vdbg is defined as :
 	   # define EXTRA_FMT "%s: "
 	   # define EXTRA_ARG ,__FUNCTION__
-	   # define llvdbg(format, ...) \
-	   lowsyslog(LOG_DEBUG, EXTRA_FMT format EXTRA_ARG, ##__VA_ARGS__)
+	   #  define vdbg(format, ...) \
```

Comment:
there seems extra space at #  define

Reply:
It was come from line 391 of left side. Only the location was changed. I think it was copied and was pasted from debug.h. This commit is not related with that. Let's change it with another commit if necessary.